README.md

```diff
@@ -1 +1,6 @@
-# Hello-Microverse
```

Comment:
- [ ] Kindly add a descriptive README file to your project. Please use the template provided by Microverse to customize your Readme. Here is the [Link](https://github.com/microverseinc/readme-template/blob/master/README.md) to the Template

Reply:
hello. How to add description to pull request without creating new pull request.